module_auto_update/models/module.py

```diff
@@ -48,34 +53,26 @@ def _store_checksum_installed(self, vals):
 
     @api.multi
     def button_uninstall_cancel(self):
-        return super(
-            Module,
-            self.with_context(retain_checksum_installed=True),
-        ).button_uninstall_cancel()
+        # TODO Use super() like in v10 after pull is merged
+        # HACK https://github.com/odoo/odoo/pull/18597
```

Comment:
This was fixed (see #927) and besides it never affected v10, so leave this as it was please.